historical/common/sqs.py

```diff
@@ -0,0 +1,59 @@
+"""
+.. module: historical.common.sqs
+    :platform: Unix
+    :copyright: (c) 2018 by Netflix Inc., see AUTHORS for more
+    :license: Apache, see LICENSE for more details.
+.. author:: Mike Grima <mgrima@netflix.com>
+"""
+import uuid
+
+import boto3
+
+from historical.constants import CURRENT_REGION
+
+
+def get_queue_url(queue_name):
+    client = boto3.client("sqs", CURRENT_REGION)
+    queue = client.get_queue_url(QueueName=queue_name)
+
+    return queue["QueueUrl"]
+
+
+def make_sqs_record(event):
+    """Get a dict with the components required for SQS"""
+    return {
+        "Id": uuid.uuid4().hex,
+        "DelaySeconds": 0,
+        "MessageBody": event
+    }
+
+
+def produce_events(events, queue_url):
+    """
+    Efficiently sends events to the SQS event queue.
+    """
+    client = boto3.client('sqs', region_name=CURRENT_REGION)
+
+    limiter = 10    # Max for SQS
+    current_batch = 1
+    total_batch = int(len(events) / limiter)
+    remainder = len(events) % limiter
+    offset = 0
+
+    while current_batch <= total_batch:
```

Comment:
I _think_ this could be cleaned up by re-using the `chunks` function from common.kinesis

Reply:
Submitting an update with that logic.